packages/core/src/internal/deployment-loader/file-deployment-loader.ts

```diff
@@ -90,6 +93,23 @@ export class FileDeploymentLoader implements DeploymentLoader {
     );
 
     await writeFile(buildInfoFilePath, JSON.stringify(buildInfo, undefined, 2));
+
+    const debugInfoFilePath = path.join(
+      this._paths.artifactsDir,
+      `${futureId}.dbg.json`
+    );
+
+    await writeFile(
+      debugInfoFilePath,
+      JSON.stringify(
+        {
+          _format: "hh-sol-dbg-1",
+          buildInfo: `../build-info/${buildInfo.id}.json`,
```

Comment:
This path could be calculated relatively with something like `path.relative`. That would get you an OS specific format. Is that what is expected in HH debug files?

Reply:
It's currently computed with:

```ts
    const relativePathToBuildInfo = path.relative(
      path.dirname(artifactPath),
      pathToBuildInfo
    );
```

Reply:
which would be

```ts
    const relativePathToBuildInfo = path.relative(
      this._paths.artifactsDir,
      debugInfoFilePath,
    );
```

Reply:
Awesome. Change applied.